lib/send-mail.ts

```diff
@@ -7,7 +7,7 @@ export interface IParsedMBox {
     cc?: string[];
     date?: string;
     from?: string;
-    headers?: Array<{ key: string; value: string; }>;
+    headers?: Array<{ key: string; value: string }>;
```

Comment:
Technically, this change does not fit the commit message, but I do not object strongly to it being in the same commit as the other `;`-related changes.